frame/staking/src/pallet/mod.rs

```diff
@@ -1792,6 +1793,20 @@ pub mod pallet {
 			})?;
 			Ok(())
 		}
+
+		/// Sets the minimum amount of commission that each validators must maintain.
+		///
+		/// This call has lower privilege requirements than `set_staking_config` and can be called
+		/// by the `T::AdminOrigin`. Root can always call this.
+		///
+		/// Weight: `O(1)`
+		#[pallet::call_index(25)]
+		#[pallet::weight(T::WeightInfo::set_min_commission())]
+		pub fn set_min_commission(origin: OriginFor<T>, new: Perbill) -> DispatchResult {
```

Comment:
Name this `apply_min_commission` to keep it inline with `force_apply_min_commission`?

Reply:
The `force_apply_min_commission` applies `min_commission` on the provided validator which is different thing than setting the value right? Other calls more similar to this are `set_validator_count` and `set_staking_config`.